mycroft/skills/intent_service.py

```diff
@@ -554,3 +560,29 @@ def handle_entity_manifest(self, message):
         self.bus.emit(message.reply(
             "intent.service.padatious.entities.manifest",
             {"entities": self.padatious_service.registered_entities}))
+
+
+def is_old_style_keyword_message(message):
```

Comment:
should this be private since it will be dropped soon? just be sure nothing uses it in the wild

Reply:
That's a good idea. will update